imdb.py

```diff
@@ -5,24 +5,43 @@
 import bs4
 
 
```

Comment:
Write the inline documentation a line before the actual command, instead of after the command.

Reply:
like for example , the line 8 documentation should be made above  the current line 8 command??

Reply:
@akshatnitd I have done the proper documentation. 
Kindly look and suggest changes if any.
Thanks